contracts/axelar-interchain-token-service/Cargo.toml

```diff
@@ -13,5 +13,4 @@ soroban-sdk = { workspace = true }
 
 [dev_dependencies]
 soroban-sdk = { workspace = true, features = ["testutils"] }
-axelar-soroban-std = { workspace = true, features = ["testutils"] }
-
+axelar-soroban-std = { workspace = true, features = ["testutils"] }
```

Comment:
still seeing trailing newline warning, take a look at editor setting to fix this